platforms/linux/org.qlcplus.QLCPlus.appdata.xml

```diff
@@ -67,15 +67,15 @@
  </description>
  <screenshots>
   <screenshot type="default">
-   <image>http://www.qlcplus.org/shots/qlcplus_fixtures.png</image>
+   <image>https://www.qlcplus.org/user/pages/01.home/Fixtures.PNG</image>
    <caption>Fixtures panel</caption>
   </screenshot>
   <screenshot>
-   <image>http://www.qlcplus.org/shots/qlcplus_virtualconsole2.png</image>
+   <image>https://www.qlcplus.org/user/pages/01.home/VirtualConsole.PNG</image>
```

Comment:
This one is
https://www.qlcplus.org/user/pages/05.discover/01.features/10._virtual-console/qlcplus_virtualconsole2.png